lib/scrabble.rb

```diff
@@ -15,4 +13,19 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+
+  def score(word)
+    list_of_values = []
+    if word == '' || word == nil
+      list_of_values << 0
+    else
+      list_of_letters = word.upcase.chars
+      list_of_letters.each do |x|
```

Comment:
Prefer `letter` to `x` here, since `letter` actually tells us what we're working with in this block.